honeybee/_hbanalysissurface.py

```diff
@@ -78,16 +82,25 @@ def from_json(cls, srf_json):
         """Create a surface from json object.
 
         The minimum schema is:
-        {"name": "",
-        "vertices": [[(x, y, z), (x1, y1, z1), (x2, y2, z2)]],
-        "surface_type": null  // 0: wall, 5: window
-        }
+            {"name": "",
+            "vertices": [[(x, y, z), (x1, y1, z1), (x2, y2, z2)]],
+            "surface_material": {}  // radiance material json file
+            }
         """
         name = srf_json["name"]
         vertices = srf_json["vertices"]
-        type_id = srf_json["surface_type"]
-        srf_type = surfacetype.SurfaceTypes.get_type_by_key(type_id)
```

Comment:
`srf_type` is still a valid input for surface and should not be removed. In radiance it's not critical if user provides the material but in EnergyPlus simulation it is.